styles/_utils.scss

```diff
@@ -341,3 +341,29 @@ $flex-align-content: (
     align-content: $value;
   }
 }
+
+$gap: 1rem;
+$gap-values: (
+  0: 0,
```

Comment:
can we also have some more values like here https://www.primefaces.org/primeflex/gap ?
till `5em` gap

Reply:
added